src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -89,7 +89,6 @@ export default class QuestionnaireForm extends Component {
 
   getRetrieveSaveQuestionnaireUrl = () => {
     // read configuration 
-    let updateDate = new Date();
```

Comment:
this variable *is* used, should be reinstated, the `days` variable below is the one that isn't used.